app/components/modals/event-transfer-modal.js

```diff
@@ -0,0 +1,10 @@
+import { computed } from '@ember/object';
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend({
+  isSmall          : true,
+  confirmEventName : '',
+  isNameDifferent  : computed('confirmEventName', function() {
```

Comment:
You're using `eventName` too. So, you must ensure your computed property has that also as a dependant key.